crates/db_connection_pool/src/mysqlpool.rs

```diff
@@ -80,9 +84,31 @@ impl MySQLConnectionPool {
                     connection_string =
                         connection_string.tcp_port(mysql_port.parse::<u16>().unwrap_or(3306));
                 }
+                if let Some(mysql_sslmode) = params.get("mysql_sslmode") {
+                    match mysql_sslmode.as_str() {
+                        "disable" | "require" | "prefer" => {
+                            ssl_mode = mysql_sslmode.as_str();
+                        }
+                        _ => {
+                            InvalidParameterSnafu {
+                                parameter_name: "mysql_sslmode".to_string(),
+                            }
+                            .fail()?;
+                        }
+                    }
+                }
             }
         }
 
+        let default_ssl_opts = SslOpts::default().with_danger_accept_invalid_certs(true);
+        let ssl_opts = match ssl_mode {
+            "disable" => None,
+            "require" => Some(SslOpts::default()),
+            "prefer" => Some(default_ssl_opts),
+            _ => Some(default_ssl_opts),
```

Comment:
Default should be require

Reply:
We use prefer as default for postgres, should it also be changed?

Reply:
Yes

Reply:
Changed